examples/AeroSharp.Examples/Maps/MapExamples.cs

```diff
@@ -17,12 +17,15 @@ public async Task ExecuteAsync()
             new MapGetByKeyExample(_clientProvider),
             new MapCustomClassExample(_clientProvider),
             new MapRemoveByKeyExample(_clientProvider),
-            new MapDeleteExample(_clientProvider)
+            new MapDeleteExample(_clientProvider),
+            new MapGetByRankExample(_clientProvider),
+            new NestedCdtMapGetByRankExample(_clientProvider),
```

Comment:
I'd propose standardizing the casing of 'CDT'. It may also be worth just spelling out "CollectionDataType"

Reply:
I know it's dotnet, but when can a function name be too long?

Good callout though here as I think we don't really talk about CDTs anywhere even though it is contained in the AS docs

Reply:
Sorry, this selection doesn't highlight that there were two casings (another one on line 23). It doesn't have to be the longer counterpart, but we should choose between `Cdt` and `CDT` for sure.

Reply:
Oh! Definitely!

Reply:
My personal preference is to ditch this acronym. Maybe I'm naïve, but this is the first time I've seen the "CDT" acronym for collections.